ProjectMarsLanguageFeature/Support/DeleteAnExistingLanguage.cs

```diff
@@ -6,7 +6,7 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace ProjectMarsLanguageFeature.Utilities
```

Comment:
all this support functionality should live in pages as it uses webdriver to find elements, the support should be used for waits and doing somthing generic

Reply:
I will take care of it.